assignments/objects.js

```diff
@@ -15,31 +15,87 @@ const example = {
   "name": "Example",
   "email": "examples@you.edu",
   "gender": "F"
-}
+};
 
 // Write your intern objects here:
 
+const int1 = {
+  id: 1,
+  name: "Mitzi",
+  email: "mmelloy0@psu.edu",
+  gender: "F"
+};
+
+const int2 = {
+  id: 2,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M"
+};
+
+const int3 = {
+  id: 3,
+  name: "Keven",
+  email: "kmummery2@wikimedia.org",
+  gender: "M"
+};
+
+const int4 = {
+  id: 4,
+  name: "Gannie",
+  email: "gmartinson2@illinois.edu",
+  gender: "M"
+};
+
+const int5 = {
+  id: 5,
+  name: "Antonietta",
+  email: "adaine5@samsung.com",
+  gender: "F"
+};
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(int1.name);
+
 // Kennan's ID
 
+console.log(int2.id);
+
 // Keven's email
 
+console.log(int3.email);
+
 // Gannie's name
 
+console.log(int4.name);
+
+
 // Antonietta's Gender
 
+console.log(int5.name);
+
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+int2.speak = function () {
+  return ('Hello, my name is ' + this.name);
+}
+console.log(int2.speak());
+
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
 
+int5.multiplyNum = function (num1, num2) {
```

Comment:
Creating a new property from outside of the object, Love it!!! You haven't learned this yet, but you still killed it!!